AuroraJobEx.sh

```diff
@@ -5,4 +5,4 @@
 #SBATCH --mem-per-cpu=6000 #This tells the system how much memory should be used per CPU involved in the calculation of the job 
 #SBATCH -N 2 #This specifies how many nodes should be used for the job 
```

Comment:
Do you need two full nodes (40 cores) to be reserved for your job? I would have thought you could do single-core, or is this multithreaded in some way?

Reply:
I'll merge this in now, but this question still stands; just want to check that you aren't requesting more resources than you nee since that leads to unnecessary waiting time for both you and others